aioca/cache.py

```diff
@@ -0,0 +1,131 @@
+import asyncio
+
+from . import camonitor, CANothing
+
+
+class PVCache:
```

Comment:
This will need a bit of work to shoehorn typing into it. I guess something like this might work:
```suggestion
from typing import TypeVar
T = TypeVar("T")
V = TypeVar("V", Dict[str, T], List[T])
class PVCache(Generic[V[str]]):
...
    def get_nowait(self) -> V[AugmentedValue]:
...
```
This says if you pass a List[str] as PVs, then you will get a List[AugmentedValue] from get_nowait, and if you pass a Dict[str, str] as PVs you will get a Dict[str, AugmentedValue].
It might work, depends on what mypy supports...